server/src/main/java/org/opensearch/index/store/remote/directory/RemoteSnapshotDirectory.java

```diff
@@ -35,6 +37,9 @@
  * @opensearch.internal
  */
 public final class RemoteSnapshotDirectory extends Directory {
+
+    public static final Version SEARCHABLE_SNAPSHOT_EXTENDED_COMPATIBILITY_MINIMUM_VERSION = LegacyESVersion.fromId(6000099);
```

Comment:
Happy to hear feedback on a better place to stick this. Ideally, this should be modeled as a part of the `FeatureFlag` but that would require a broader change to allow feature "flags" to support values beyond just `boolean`.

Reply:
I think it is fine not being a part of the feature flag. This place seems as good as any to me as far as where to put it now.